media-gfx/apngopt/files/apngopt-1.4-r1-makefile.patch

```diff
@@ -21,7 +21,7 @@ Use system zopfli library and respect CFLAGS.
  
  $(PACKAGE) : objdirs $(OBJECTS)
 -	$(CC) -o $@ $(OBJECTS) -s $(LIBS)
-+	$(CC) -o $@ $(OBJECTS) $(LIBS)
++	$(CC) -o $@ $(OBJECTS) $(CFLAGS) $(LDFLAGS) $(LIBS)
```

Comment:
those two lines seem wrong... what if you coalesce them into one:
```suggestion
+	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $(OBJECTS) $(LIBS)
```
remember, `LDFLAGS` has to come very early in the link line, since it only affects stuff that comes after it